demos/animation/anim.go

```diff
@@ -60,8 +60,12 @@ func (*DefaultScene) Type() string { return "GameWorld" }
 func (*DefaultScene) CreateEntity(point *engo.Point, spriteSheet *engo.Spritesheet) *Animation {
 	entity := &Animation{BasicEntity: ecs.NewBasic()}
 
-	entity.SpaceComponent = engo.SpaceComponent{*point, 150, 150}
-	entity.RenderComponent = engo.NewRenderComponent(spriteSheet.Cell(0), engo.Point{3, 3})
+	entity.SpaceComponent = engo.SpaceComponent{
+		Position: *point,
+		Width:    150,
+		Height:   150,
+	}
+	entity.RenderComponent = engo.RenderComponent{Drawable: spriteSheet.Cell(0), Scale: engo.Point{3, 3}}
```

Comment:
I will, as long as we set more than 1 variable. If we only set 0 or 1 variables, I'd rather keep it a one-liner.